packages/sql-mapper/mapper.js

```diff
@@ -120,6 +120,8 @@ async function connect ({ connectionString, log, onDatabaseLoad, poolSize, inclu
   let queries
   const { db, sql } = await createConnectionPool({ log, connectionString, poolSize, queueTimeoutMilliseconds, acquireLockTimeoutMilliseconds, idleTimeoutMilliseconds })
 
+  db.connectionString = connectionString
```

Comment:
Is this needed?

Reply:
...mmm...no. We can also read it from the config. Changing this.